src/client/clientlauncher.cpp

```diff
@@ -228,11 +231,14 @@ bool ClientLauncher::run(GameStartData &start_data, const Settings &cmd_args)
 				core::rect<s32>(0, 0, 10000, 10000));
 
 			bool game_has_run = launch_game(error_message, reconnect_requested,
```

Comment:
I feel at this point the three `reconnect` parameters might deserve their own `struct`? It seems to me this would make the code somewhat less verbose.